### DISCUSSION TITLE
Change Button Colours?

### DISCUSSION THREAD

**Discussion by pbyrne-ms** [dbuezas/lovelace-plotly-graph-card#134] [Q&A] [answered]
Hi - love your card, thanks so much for creating it! I was wondering if there is a way to change the Range selector button colours?
The default silver/grey colors blend into the background of the particular HA theme and color scheme I use. I see on the Plotly website it is possible to do this if using natively in javascript, but I can't figure out how to do it in the card YAML. If this is possible, could you show an example of how to change the button background color and button text? Thanks again!

**Comment by dbuezas** [comment, marked as answer]
Happy to hear that!

Here's how to style them: https://plotly.com/javascript/reference/#layout-xaxis-rangeselector-bgcolor

E.g: 
```yaml
layout:
  xaxis:
    rangeselector:
      bgcolor: red
```

**Comment by pbyrne-ms** [comment]
Thanks, David, works great and that has joined the dots for me to translate some other things I see on the plotly site to YAML syntax, cheers!
